test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -503,7 +531,8 @@ def forward(self, input):
         self.run_test_with_fx_to_onnx_exporter_and_onnx_runtime(
             ViewModel(),
             (x,),
-            # additional_test_inputs=[((y,),)],
+            # additional_test_inputs=[((y,),)],  # TODO: Without `additional_test_inputs` arg, dynamic shape cannot be verified
+            skip_dynamic_shapes_check=True,
```

Comment:
why skip, is it failing?

Reply:
https://github.com/pytorch/pytorch/pull/105950 disabled `additional_test_inputs` when Edward from Meta realized this test does not generate dynamic shape (after recent changes on dynamo)

For some reason we didn't find a new test for the zero dim case nor deleted the no-op test, leaving us with a `dynamic_zero_dim` that generates a `static_zero_dim` :)

Reply:
thx I recall now, let's add comment that this exports as static shape even when dynamic_shapes=True due to 0/1 specialization